docker-compose-net/docker-compose-net.md

```diff
@@ -0,0 +1,234 @@
+
+The following topics are discussed: 
+
+  * [What You Will Use](#what-use)
+  * [Before You Begin](#before)
+  * [Attaching Containers to a Weave Network](#attaching-containers)
+  * [Docker Compose and the Weave Net Docker API Proxy](#setup-proxy)
+     * [Docker Compose File Syntax for the Weave Docker API Proxy](#compose-proxy)
+  * [Docker Compose and the Weave Net Docker Plugin](#setup-plugin)
+    * [Docker Compose File Syntax and the Weave Net Docker Plugin](#compose-plugin)
+
+##<a name="what-use"></a>What You Will Use
+
+* [Weave](http://weave.works)
+* [Weave Scope](http://weave.works/scope/index.html)
+* [Docker](http://docker.com)
+* [Docker Compose](https://www.docker.com/docker-compose)
+
+##<a name="before"></a>Before You Begin
+
+If you are using Windows, you can install [Docker for Windows](https://docs.docker.com/engine/installation/windows/), and if you are on a Mac, install [Docker for Mac](https://docs.docker.com/engine/installation/mac/) which provides all of the tools you need to complete this guide.
+
+For other operating systems, please install and configure the following separately before proceeding:
+
+  - [`docker-machine`](http://docs.docker.com/machine/#installation) binary (_`>= 0.8.0`_)
+  - [`docker`](https://docs.docker.com/installation/#installation) binary, at least the client (_`>= v1.6.x`_)
+  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (_`>= 4.3.x`_)
+  - `curl` (_any version_)
+
+##<a name="attaching-containers"></a>Docker Compose and Attaching Containers to a Weave Network 
+ 
+ There are two different ways that you can attach containers to a Weave network when using Compose (which method to use is 
+ entirely up to you):
+ 
+ **1.** The Weave Net Docker API Proxy. See [Integrating Docker via the API Proxy](https://www.weave.works/docs/net/latest/weave-docker-api/).  
+ 
+ **2.**  The Docker Network Plugin framework. The Docker Network Plugin is used when 
+ Docker containers are started with the --net flag, for example: 
+ 
+ `docker run --net <docker-run-options>`
+ 
+ **Where,** 
+ 
+  * `<docker-run-options>` are the [docker run options](https://docs.docker.com/engine/reference/run/) 
+  you give to your container on start 
+ 
+ >>**Note:** If a Docker container is started with the `--net` flag, then the Weave Docker API Proxy
+ is automatically disabled and is not used to attach containers. See [Integrating Docker via the Network Plugin](https://www.weave.works/docs/net/latest/features/#plugin).
+
+
+##<a name="setup-proxy"></a>Docker Compose and the Weave Net Docker API Proxy
+
```

Comment:
Is the goal here to install Weave on the VMs, but then run `docker-compose` from the user's machine? Or to run `docker-compose` on the VMs? It's not clear currently. Would be good to add a sentence here like
* "The goal is to log into and install Weave on the VMs, and then run `docker-compose` from your workstation, pointed at the Proxy running on the VMs."
* "The goal is to log into and install Weave on the VMs, and then run `docker-compose` in the VMs."

I think the first option would be more realistic/useful to users, because they'll have their compose files on their workstation.